tracker/data/processing.py

```diff
@@ -573,11 +619,15 @@ def https_behavior_for(name, pshtt, sslyze, parent_preloaded=None):
         else:
             bod_crypto = 1
 
+
+        uses_accepted_ciphers = all(cipher in ACCEPTED_CIPHERS for cipher in sslyze.get("Accepted Ciphers").split(', '))
```

Comment:
This is fine, just curious if there's a way to also populate the list of 'unapproved ciphers', even if it's just something we dump out via the CSV export or something. I'd rather have more data in the DB then less.

Reply:
Yeah, that shouldn't be difficult. I can make a change to that effect